packages/components/src/components/hds/layout/grid/types.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+export enum HdsLayoutGridAlignValues {
+  Start = 'start',
+  Center = 'center',
+  End = 'end',
+  Stretch = 'stretch',
+}
+
+export type HdsLayoutGridAligns = `${HdsLayoutGridAlignValues}`;
+
+export enum HdsLayoutGridGapValues {
+  'Four' = '4',
+  'Eight' = '8',
+  'Twelve' = '12',
+  'Sixteen' = '16',
+  'Twentyfour' = '24',
```

Comment:
These was changed to camel case in the Flex PR so it should probably match.
```suggestion
  'TwentyFour' = '24',
```

Reply:
(same for thirty two and forty eight)

Reply:
Thanks! Updating all together locally & pushing up.